src/utilities/countries.js

```diff
@@ -1 +1,3 @@
 export default [{ id: 1, name: 'Northern Ireland' }, { id: 2, name: 'Mexico' }];
+
+export const countryText = [null, 'Northern Ireland', 'Mexico'];
```

Comment:
can you add a new line at the bottom here?

Also not mad about an array which is based on having countries in the right order - if this got an bigger it would be tough to maintain

Reply:
I'll do that just for you Dan.

Any suggestions around the array thing? I only get an ID to work with from the back-end so there's always going to be maintenance to keep the ID and description text paired.

Reply:
right now I'm thinking an object, something like

`export const countryText = {
  1: 'Northern Ireland',
  2: 'Mexico'
}`?